<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget_tabtext">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="QMenu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="Qaction_open"/>
    <addaction name="Qaction_save"/>
   </widget>
   <widget class="QMenu" name="QMenu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionnone"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="Qaction_font"/>
    <addaction name="Qaction_highlight"/>
   </widget>
   <addaction name="QMenu_file"/>
   <addaction name="QMenu_help"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Qaction_open">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="Qaction_save">
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="actionnone">
   <property name="text">
    <string>none</string>
   </property>
  </action>
  <action name="Qaction_font">
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="Qaction_highlight">
   <property name="text">
    <string>高亮</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
